concepts/booleans/about.md

```diff
@@ -0,0 +1,120 @@
+# About
+
+## Booleans in Julia
+
+True or false values are represented by the `Bool` type.
+It contains only two values: `true` and `false`.
+
+```julia-repl
+julia> true
+true
+
+julia> false
+false
+
+julia> typeof(true)
+Bool
+```
+
+In contrast to several other languages, Julia deliberately has no concept of "truthiness".
+Only expressions which evaluate to `true` or `false` will be treated as a `Bool`.
+
+Specifically, empty arrays or strings will *not* be interpreted as `false`.
+There must be an appropriate test such as `isempty()` if you want special handlind for empty values.
+
+## Boolean operators
+
+There are three [Boolean operators][boolean-operators] in Julia.
+
+`&&` is Boolean "and".
+It evaluates to `true` if the expressions on *both* sides of `&&` are `true`.
+
+```julia-repl
+julia> true && true
+true
+
+julia> true && false
+false
+```
+
+`||` is Boolean "or".
+It evaluates to `true` if an expression on *either* side of `||` is `true`.
+
+```julia-repl
+julia> true || true
+true
+
+julia> false || true
+true
+```
+
+`!` is Boolean "not".
+It exchanges (inverts) `true` and `false` values.
+
+```julia-repl
+julia> !true
+false
+
+julia> !false
+true
+```
+
+These operators will be familiar to users of many other languages, though *not* Python.
```

Comment:
This sentence is confusing. `and` and `or` exist in other languages. Perhaps we say the operators and these concepts should be familiar coming from other languages but they might have different names there.